kodi_addon_checker/check_files.py

```diff
@@ -110,3 +111,16 @@ def check_file_whitelist(report: Report, file_index: list, addon_path: str):
                 report.add(Record(WARNING,
                                   "Found non whitelisted file ending in filename %s" %
                                   relative_path(os.path.join(file["path"], file["name"]))))
+
+
+def check_file_permission(report: Report, addon_path: str):
+    """Check whether the files present in addon are marked executable
+       or not
+        :addon_path: Path of the addon
+    """
+
+    files = Path(addon_path).glob('**/*.py')
```

Comment:
Shouldn't the permission be checked for all files?
Anyone can imagine an use case where a non python file has to be marked executable?

Reply:
No, your absolutely right.